### ISSUE TITLE
ecr-viewer lab info section missing visual indicators on accordion buttons

### ISSUE THREAD

**Issue by BobanL** [CDCgov/phdi#1753]
## Action Requested
- Add visual focus indicator on lab result accordion buttons
![image](https://github.com/CDCgov/phdi/assets/10108172/97096122-860c-4018-853e-7024a91bd534)

## Acceptance Criteria
- Visual focus indicator is on each accordion button in lab results.

## Additional Information
- https://cdc-devtools.dequecloud.com/issues/ad946c8a-5bfc-4a72-b0e5-8265d37aa3e0
- https://docs.deque.com/issue-help/1.0.0/en/focus-indicator-missing
- Example of missing indicator (those accordion header buttons should have focus indicators).
https://github.com/CDCgov/phdi/assets/10108172/43da25ad-da1d-453b-a641-b301a49acec7
